src/components/AnchorForCommentsOnly/anchorForCommentsOnlyPropTypes.js

```diff
@@ -15,24 +15,30 @@ const propTypes = {
         This is unused in native, but is here for parity with web */
     target: PropTypes.string,
 
-    /** Should the link be treated as a file download or a regular hyperlink? (relevant to native platforms only)  */
-    shouldDownloadFile: PropTypes.bool,
+    /** Flag to differentiate attachments and hyperlink. Base on flag link will be treated as a file download or a regular hyperlink?
+     * (relevant to native platforms only)  */
```

Comment:
Remove the ? and `(relevant to native platforms only)` since it applies to all platforms

Reply:
Done